src/CONST.ts

```diff
@@ -1463,6 +1463,12 @@ const CONST = {
     },
     QUICKBOOKS_ONLINE: 'quickbooksOnline',
 
+    QUICKBOOKS_DESKTOP_CONFIG: {
+        MARK_CHECKS_TO_BE_PRINTED: 'markChecksToBePrinted',
+        REIMBURSABLE_EXPENSES_ACCOUNT: 'reimbursableExpensesAccount',
+        REIMBURSABLE_EXPENSES_EXPORT_DESTINATION: 'reimbursableExpensesExportDestination',
```

Comment:
Same here it should be in `export.reimbursable`.

Reply:
I'm raising a backend PR to fix this, it'll resolve the incorrect setting name error.

Reply:
You need to pass down the `enabled` param and set it to `true`, otherwise it defaults to false.

Reply:
Yes you can pass `true` or `false`, if you don't pass down the `enabled` param it will be `false` by default.

Reply:
As I've said earlier, all the new API commands are listed [here](https://docs.google.com/document/d/1LQNlTBlHjFqRdru_4UkqVr0QRseNNDq2IhFFpXcgfwY/edit?tab=t.0#bookmark=id.3m18ad5ajir9).

![image](https://github.com/user-attachments/assets/7d67483f-c093-4bbd-9b44-b8d0d5b9b30e)

There's no new API command called `UpdateQuickbooksDesktopReimbursableAccount`, you should do like for QBO and call the generic API commands to update policy's connection configuration.

Reply:
My bad, previously QBO called the generic commands to update (`UpdatePolicyConnectionConfiguration`, `UpdateManyPolicyConnectionConfigurations`). Now it uses dedicated API commands, I'll create them.

In the meantime, you can use the generic API commands to advance.

Reply:
Thanks, PR to create all these specific API commands should be ready tomorrow. I'll give you an update.